packages/angular/src/tracing.ts

```diff
@@ -83,6 +92,7 @@ export class TraceService implements OnDestroy {
       }
 
       if (activeTransaction) {
+        this._activeTransaction = activeTransaction;
```

Comment:
Well and there I was thinking I found a smart workaround. So the only reason I added this member variable was because at the beginning it seemed like `getActiveTransaction` returned undefined. I just tested this again and ... dunno what changed but apparently now it works. So you're right, no need for `_activeTransaction`.

Reply:
[23242aa](https://github.com/getsentry/sentry-javascript/pull/5416/commits/23242aa88d02054c0f128c04389e0da476cc77ea)